nml/actions/real_sprite.py

```diff
@@ -296,8 +296,8 @@ def __init__(self, mapping, label = None, poslist = None):
 
     def debug_print(self, indentation):
         generic.print_dbg(indentation, 'Recolour sprite, mapping:')
-        for assignment in self.mapping:
-            generic.print_dbg(indentation + 2, '{}: {};'.format(assignment.name, assignment.value))
+        for existing_assignment in self.mapping:
+            generic.print_dbg(indentation + 2, '{}: {};'.format(existing_assignment.name, existing_assignment.value))
```

Comment:
What's the purpose of these changes (more below)?

They add verbosity without making anything clearer.

Reply:
> \<LordAro\> FLHerne: there's an import named assignment, so fixing variable shadowing

Could we use 'mapped' instead? It would be shorter. :p

Reply:
by 2 characters :p but sure